eessi_bot_event_handler.py

```diff
@@ -588,18 +588,23 @@ def handle_bot_command_status(self, event_info, bot_command):
 
         comment_status = ''
         comment_status += "\nThis is the status of all the `bot: build` commands:"
-        comment_status += "\n|arch|result|date|status|url|"
-        comment_status += "\n|----|------|----|------|---|"
+        comment_status += "\n|on|for|repo|result|date|status|url|"
+        comment_status += "\n|----|----|----|------|----|------|---|"
         for x in range(0, len(status_table['date'])):
-            comment_status += f"\n|{status_table['arch'][x]}|"
+            comment_status += f"\n|{status_table['on arch'][x]}|"
+            comment_status += f"{status_table['for arch'][x]}|"
+            comment_status += f"{status_table['for repo'][x]}|"
             comment_status += f"{status_table['result'][x]}|"
             comment_status += f"{status_table['date'][x]}|"
             comment_status += f"{status_table['status'][x]}|"
             comment_status += f"{status_table['url'][x]}|"
 
         self.log(f"Overview of finished builds: comment '{comment_status}'")
         issue_comment = create_comment(repo_name, pr_number, comment_status, ChatLevels.MINIMAL)
-        return issue_comment
+        if issue_comment:
+            return f"\n  - added status comment {issue_comment.html_url}"
+        else:
+            return "\n  - failed to create status comment"
```

Comment:
Not sure why this used to return an IssueComment instance. Looking at the other bot command handlers, it makes sense to me that this also returns a string. Can you update the docstring of the function accordingly?

Reply:
Good point, done. It was definitely a wrong type before, it gave me a hard error.